frontend_lib/src/component/Tags/Tag.jsx

```diff
@@ -3,8 +3,6 @@ import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
 import IconButton from '../Button/IconButton.jsx'
 
-require('./TagList.styl')
-
```

Comment:
Can you replace 
```require('./TagList.styl')```
by
```// require('./TagList.styl')  // see https://github.com/tracim/tracim/issues/1156```
Instead of removing it?

Reply:
Updated